metrics_gather.sh

```diff
@@ -250,7 +250,7 @@ av2 | av2-ai | av2-ra | av2-ra-st | av2-ld | av2-as)
       echo "#!/bin/bash" > /tmp/enc$$.sh
       echo "TIMER='time -v --output='enctime$$-\$1.out" >> /tmp/enc$$.sh
       echo "RUN='$AOMENC --codec=av1 --cq-level=$x --test-decode=fatal $CTC_PROFILE_OPTS -o $BASENAME-'\$1'.obu $EXTRA_OPTIONS --limit=130 --'\$1'=65 $FILE'" >> /tmp/enc$$.sh
-      echo "\$(\$TIMER \$RUN > $BASENAME$$-stdout.txt)" >> /tmp/enc$$.sh
+      echo "\$(\$TIMER \$RUN > $BASENAME$$-stdout.txt 2> $BASENAME$$-stderr.txt)" >> /tmp/enc$$.sh
```

Comment:
Doesn't $BASENAME$$-stderr.txt contain the PID of the executing process (for use with GOP parallelism)?

It looks like the files are copied back, but they will still contain the PID and not easily linkable from the AWCY frontend. Are these merged somewhere?

Reply:
No, they shouldn't contain the PID? However, they don't actually get merged properly. Both of those should probably be >> for mt.

Reply:
The $$ in those file names is the PID. I don't think you want to use >> as that would interleave the output. Instead, after both have completed, you should merge these into a single file before copying.